lib/resources/r4/charge_item_modify.yaml

```diff
@@ -0,0 +1,417 @@
+---
+name: Parameters
+field_name_base_url: https://hl7.org/fhir/r4/parameters-definitions.html#Parameters
+fields:
```

Comment:
Is it possible to have one section for Parameters and one section for ChargeItem? I'm not sure if that is possible or will be formatted correctly but it would be nice to have examples of both parameters and the ChargeItem subfields not mixed in. (Just a nice to have.)

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/780/commits/8854164a3591fb59fce490a6fae6aa0e29c11429#diff-3c565d9c41e2bb3d417efe63a774683e72b07cd38f2ff54a695c695999f29938R25-R42

Reply:
I think having this is good, a few notes:

- Let's group the transactionId parameter up top with this one as well.
- Let's structure this example the same way we do for transacationId.
- Let's change the url to be a Parameters.parameter link.
- Let's add a note specifying that resource is required for this parameter and that it must be a ChargeItem.
- I'm not sure we need the 'only one' note since we don't have the same one on transactionId.

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/780/commits/0e5c3b242a28011c947985372687f1784aea06ba#diff-3c565d9c41e2bb3d417efe63a774683e72b07cd38f2ff54a695c695999f29938R32-R63